src/services/synapse.service.ts

```diff
@@ -17,7 +18,8 @@ export interface SynapseService {
   ): Promise<SynapseSyncResponse>;
   fetchRoomIdByName(
     name: string,
-  ): Promise<{ room_id: string; servers: string[] }>;
+    accessToken: string | undefined,
```

Comment:
This function should return only one room by definition.
We should not change the behavior. IF that's the case, we should change the name